google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -1229,54 +1223,63 @@ def read(
                 )
             ]
 
-        return self._load_read_feature_values_response_to_dataframe(response=response)
+        return _load_read_feature_values_response_to_dataframe(response=response)
 
-    @staticmethod
-    def _load_read_feature_values_response_to_dataframe(
-        response: Union[
-            gca_featurestore_online_service.ReadFeatureValuesResponse,
-            List[gca_featurestore_online_service.ReadFeatureValuesResponse],
-        ],
-    ):
-        """Loads read_feature_values_response to dataframe
 
-        Args:
-            response (Union[featurestore_online_service.ReadFeatureValuesResponse,
-                            List[featurestore_online_service.ReadFeatureValuesResponse]):
-                Required. featurestore_online_service.ReadFeatureValuesResponse from `read_feature_values`
-                or Iterable in list from `streaming_read_feature_values`.
-        Returns:
-            pd.DataFrame - response in DataFrame
+def _load_read_feature_values_response_to_dataframe(
```

Comment:
What was the reasoning for not keeping this a staticmethod? It seems like it's coupled to the class.

Reply:
changed to staticmethod